src/SprykerMiddleware/Zed/Process/Dependency/Plugin/Stream/OptionAwareStreamPluginInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerMiddleware\Zed\Process\Dependency\Plugin\Stream;
+
+interface OptionAwareStreamPluginInterface
```

Comment:
I understand correctly: we do not do Extension modules in middleware, right? We do not do specifications on plugin interfaces either, right?

Reply:
I added short specification with plugin description